### ISSUE TITLE
Revisit wording related to clinical use

### ISSUE THREAD

**Issue by jcfr** [Slicer/Slicer#7208]
>  "Slicer is NOT approved for clinical use" may be interpreted like it was rejected. 
>
> We could say that developers of 3D Slicer did not seek regulatory approval of the freely distributed application for any particular clinical use.

Occurrences of test to update:
* https://github.com/Slicer/Slicer/blob/ff5700d9d52c486821f937b2208f8e2f28cdb539/Base/QTCore/qSlicerCoreApplication.cxx#L1795-L1804
* https://github.com/Slicer/Slicer/blob/ff5700d9d52c486821f937b2208f8e2f28cdb539/Docs/user_guide/about.md#L188
* https://github.com/Slicer/slicer.org/blob/7090f5258e20410fd63070f2b4ecdd1b9f829e4f/commercial-use.markdown#L28

_Originally posted by @lassoan in https://github.com/Slicer/Slicer/pull/7199#discussion_r1309161662_

**Comment by cpinter** [comment]
I agree. The wording seemed too harsh for the reality it tries to explain. I like the suggested alternative too.
